Dockerfile

```diff
@@ -15,10 +15,12 @@ ENV PYTHONUNBUFFERED=1
 
 COPY --from=builder /app .
 
+# Install dependencies
 RUN pip install --upgrade pip && \
     pip install -r requirements.txt
```

Comment:
Ensure that the `requirements.txt` file is present in the context of the Docker build. If it's not copied in the first stage, the `pip install -r requirements.txt` command will fail.